zipper/broadcast/broadcast_group.go

```diff
@@ -192,6 +197,8 @@ func (bg *BroadcastGroup) Fetch(ctx context.Context, request *protov3.MultiFetch
 	}
 
 	requests := bg.SplitRequest(ctx, request)
+	// TODO(gmagnusson): WAIT, HOW MANY METRICS WAS THAT
```

Comment:
Btw, zap supports sampling of the logs - you can specify how many messages per second (max) for particular logger you want to have.

It might be a good idea for future improvement to implement an HTTP Handle that allows you to temporary bump loglevel and to control sampling rate.